# Disable verification of certain binary outputs against third_party/win_build_output.
# The referenced binaries are pruned during binary pruning

--- a/build/toolchain/win/midl.py
+++ b/build/toolchain/win/midl.py
@@ -211,23 +211,6 @@ def main(arch, outdir, dynamic_guid, tlb
 
     for f in os.listdir(tmp_dir):
       ZapTimestamp(os.path.join(tmp_dir, f))
-
-    # Now compare the output in tmp_dir to the copied-over outputs.
-    diff = filecmp.dircmp(tmp_dir, outdir)
-    if diff.diff_files:
-      print 'midl.exe output different from files in %s, see %s' \
-          % (outdir, tmp_dir)
-      for f in diff.diff_files:
-        if f.endswith('.tlb'): continue
-        fromfile = os.path.join(outdir, f)
-        tofile = os.path.join(tmp_dir, f)
-        print ''.join(difflib.unified_diff(open(fromfile, 'U').readlines(),
-                                           open(tofile, 'U').readlines(),
-                                           fromfile, tofile))
-      delete_tmp_dir = False
-      print 'To rebaseline:'
-      print '  copy /y %s\* %s' % (tmp_dir, source)
-      sys.exit(1)
     return 0
   finally:
     if os.path.exists(tmp_dir) and delete_tmp_dir:
--- a/build/win/message_compiler.py
+++ b/build/win/message_compiler.py
@@ -117,23 +117,6 @@ def main():
         header_contents += sorted(define_block, key=lambda s: s.split()[-1])
       with open(header_file, 'wb') as f:
         f.write(''.join(header_contents))
-
-    # mc.exe invocation and post-processing are complete, now compare the output
-    # in tmp_dir to the checked-in outputs.
-    diff = filecmp.dircmp(tmp_dir, source)
-    if diff.diff_files or set(diff.left_list) != set(diff.right_list):
-      print 'mc.exe output different from files in %s, see %s' % (source,
-                                                                  tmp_dir)
-      diff.report()
-      for f in diff.diff_files:
-        if f.endswith('.bin'): continue
-        fromfile = os.path.join(source, f)
-        tofile = os.path.join(tmp_dir, f)
-        print ''.join(difflib.unified_diff(open(fromfile, 'U').readlines(),
-                                           open(tofile, 'U').readlines(),
-                                           fromfile, tofile))
-      delete_tmp_dir = False
-      sys.exit(1)
   except subprocess.CalledProcessError as e:
     print e.output
     sys.exit(e.returncode)
